src/main/java/mate/academy/spring/controller/ShoppingCartController.java

```diff
@@ -0,0 +1,46 @@
+package mate.academy.spring.controller;
+
+import mate.academy.spring.mapper.DtoResponseMapper;
+import mate.academy.spring.model.ShoppingCart;
+import mate.academy.spring.model.dto.response.ShoppingCartResponseDto;
+import mate.academy.spring.service.MovieSessionService;
+import mate.academy.spring.service.ShoppingCartService;
+import mate.academy.spring.service.UserService;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PutMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+@RequestMapping("/shopping-carts")
+public class ShoppingCartController {
+    private ShoppingCartService shoppingCartService;
+    private DtoResponseMapper<ShoppingCartResponseDto, ShoppingCart> shoppingCartResponseMapper;
+    private MovieSessionService movieSessionService;
+    private UserService userService;
+
+    public ShoppingCartController(ShoppingCartService shoppingCartService,
+                                  DtoResponseMapper<ShoppingCartResponseDto, ShoppingCart>
+                                          shoppingCartResponseMapper,
+                                  MovieSessionService movieSessionService,
+                                  UserService userService) {
+        this.shoppingCartService = shoppingCartService;
+        this.shoppingCartResponseMapper = shoppingCartResponseMapper;
+        this.movieSessionService = movieSessionService;
+        this.userService = userService;
+    }
+
+    @PutMapping("/movie-sessions")
+    public void addMovieSession(@RequestParam Long userId,
+                                @RequestParam Long movieSessionId) {
+        shoppingCartService.addSession(movieSessionService.get(movieSessionId),
```

Comment:
should this method be void?If we follow the rules that all methods return something to us as a result of execution, but I think the mentor will tell us exactly.

Reply:
It is better that this method return some result. But it is not a big problem for this task. So you can leave this version.